convert.py

```diff
@@ -0,0 +1,23 @@
+'''
+One of script to convert JSON structure from old to new required
+to support docstrings and C# escaped strings @"
+'''
+import json
+
+raw = open('languages.json').read()
+languages = json.loads(raw)
+
+converted = {}
+
+for key, value in languages.iteritems():
+    newquote = []
+    for quote in value['quotes']:
```

Comment:
It doesn't really matter for a one-shot conversion script, but for the record this whole loop could be rewritten as
```
value['quotes'] = [ { 'start': quote[0], 'end': quote[1] } for quote in value['quotes'] ]
```

Reply:
Oh I forgot to remove this file. I was only keeping it around while I kept converting.

Reply:
Removed.